kubectl-cloudflow/config/k8s_config_test.go

```diff
@@ -477,6 +490,37 @@ func Test_validateConfigVolumes(t *testing.T) {
 	}`)
 	assert.Empty(t, validateConfig(volumeMountPodWellformed2, spec))
 
+	volumeMountPodWellformed3 := newConfig(`
+	cloudflow.runtimes.flink.kubernetes.pods.pod {
+		volumes {
+			foo {
+				pvc {
+					name = mysecret
```

Comment:
mysecret? maybe better to use mypvc or something

Reply:
yep